### ISSUE TITLE
Tramsformer Encoder Block

### ISSUE THREAD

**Issue by s26435** [Science-Club-UG-BioAI/Nightingale#20]
Napisz pojedyńczy blok Transformera do Encodera:

1. Konstruktor:
* Argumenty: `d_model: int`, `nhead: int`, `dim_feedforward: int`, `dropout: float = 0.1`
* Co robi: 
   * tworzy multihead attention : `nn.MultiheadAttention(d_model, nhead, dropout=dropout, batch_first=True)`
   *  tworzy sequential: `[nn.Linear(d_model, dim_feedforward), nn.ReLU(), nn.Dropout(dropout), nn.Linear(dim_feedforward, d_model)]`
   * tworzy dwie warstwy `nn.LayerNorm`
   * tworzy dropout
3. Forward:
* argumenty: `x: torch.Tensor`
* Co robi: 
   *  przechodzi przez atencję -> dropout +  rezyduum -> norm1 -> sequential -> dropout +  rezyduum -> norm2 -> return
